client/src/modules/cash/modals/select-cashbox-modal.ctrl.js

```diff
@@ -43,6 +43,27 @@ function SelectCashboxModalController(Session, Instance, Cashboxes, $stateParams
       })
       .catch(Notify.handleError)
       .finally(toggleLoadingIndicator);
+
+    /**
+    * This section makes it possible to check if the user does not have permissions to a cash register or that it does not exist
+    */ 
+    Cashboxes.read(undefined, { is_auxiliary : 1 })
```

Comment:
It would be best not to download the cashboxes twice.  Is there any way to send something like an HTTP parameter `includeUserPrivileges=1`  that would do a `cash_box LEFT JOIN cashbox_permission`?  That way you could loop through the list twice and group by project and user?